src/lib/interceptor/read-many-request.interceptor.ts

```diff
@@ -74,12 +74,17 @@ export function ReadManyRequestInterceptor(crudOptions: CrudOptions, factoryOpti
             if (error) {
                 throw new UnprocessableEntityException(error);
             }
+
+            if (transformed.type === PaginationType.CURSOR && transformed.nextCursor && !transformed.query) {
+                transformed.query = 'e30=';
+            }
```

Comment:
It seems `e30=` is a magic string. How about rather using `btoa('{}')`?

Reply:
good idea